controller/model/edge_service_manager.go

```diff
@@ -65,6 +65,9 @@ func (self *EdgeServiceManager) ApplyCreate(cmd *command.CreateEntityCommand[*Se
 }
 
 func (self *EdgeServiceManager) Update(entity *Service, checker fields.UpdatedFields) error {
+	if checker != nil {
+		checker = checker.RemoveFields("encryptionRequired")
```

Comment:
this is the sorta thing I would log in the controller, but it definitely would be something I'd return back through the API if we can?

Reply:
That doesn't make sense. Why would we return anything on this?

Reply:
return back meaning: "hey, you asked us to patch this thing, but you sent in a field that we aren't gonna patch". it's confusing to use an API, issue this patch and have it succeed. This PR makes me think that sort of update would silently succeed. sure, someone can just go 'read the docs' but it's more friendly to doc it well, and return a code that says "it mostly worked". like a 206

Reply:
That will break clients looking for anything else as 206 is not in the spec.

If you are willing to break backward compact, it is better to simply remove it from the spec and cause a code-level break.